inc/Integrations/SalesforceD2C/SalesforceD2CIntegration.php

```diff
@@ -105,9 +99,9 @@ private static function get_queries( SalesforceB2CDataSource $data_source ): arr
 				'data_source' => $data_source,
 				'endpoint' => function ( array $input_variables ) use ( $base_endpoint, $service_config ): string {
 					return sprintf(
-						'%s/search/shopper-search/v1/organizations/%s/product-search?siteId=RefArchGlobal&q=%s',
+						'%s/services/data/v63.0/commerce/webstores/%s/search/products?searchTerm=%s',
```

Comment:
Any insight on the `v63.0` URL here? I'm just curious if this is a rapidly evolving API with docs for each version, or something different.

Reply:
As discussed, this is to be clarified but it seems like it's the minimum version in which this API was made available?